src/redux.js

```diff
@@ -1,3 +1,5 @@
+import _ from "lodash";
```

Comment:
As discussed, it might be good to decide whether we still need to use lodash given all the latest JS functions.  It's fine to continue using it, but it seems worth making a conscious decision to only use it when it's better than internal JS functionality.

Reply:
I removed all the references to lodash. Filter does the trick and doesn't mutate the og array.